--- www.conf.orig	2016-08-30 10:28:09.309030501 +0900
+++ www.conf	2016-08-30 10:33:17.482030501 +0900
@@ -21,9 +21,9 @@
 ; Note: The user is mandatory. If the group is not set, the default user's group
 ;       will be used.
 ; RPM: apache Choosed to be able to access some dir as httpd
-user = apache
+user = nginx
 ; RPM: Keep a group allowed to write in log dir.
-group = apache
+group = nginx
 
 ; The address on which to accept FastCGI requests.
 ; Valid syntaxes are:
@@ -35,7 +35,7 @@
 ;                            (IPv6 and IPv4-mapped) on a specific port;
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
-listen = 127.0.0.1:9000
+listen = {{ PHP_VAR_DIR }}/run/php-fpm/php-fpm.sock
 
 ; Set listen(2) backlog.
 ; Default Value: 511
@@ -45,8 +45,8 @@
 ; permissions must be set in order to allow connections from a web server.
 ; Default Values: user and group are set as the running user
 ;                 mode is set to 0660
-;listen.owner = nobody
-;listen.group = nobody
+listen.owner = nginx
+listen.group = nginx
 ;listen.mode = 0660
 
 ; When POSIX Access Control Lists are supported you can set them using
