--- Handler.php.orig	2016-08-29 16:47:44.007989541 +0900
+++ Handler.php	2016-08-29 16:48:44.009268544 +0900
@@ -38,6 +38,15 @@
      */
     public function render($request, Exception $e)
     {
-        return parent::render($request, $e);
+      if (config('app.debug'))  {
+        $whoops = new \Whoops\Run;
+        $whoops->pushHandler(new \Whoops\Handler\PrettyPageHandler());
+        return new \Illuminate\Http\Response(
+          $whoops->handleException($e),
+          $e->getStatusCode(),
+          $e->getHeaders()
+        );
+      }
+      return parent::render($request, $e);
     }
 }
