--- php-fpm.conf.orig	2016-08-30 10:28:16.857030501 +0900
+++ php-fpm.conf	2016-08-30 10:27:10.806030501 +0900
@@ -96,7 +96,7 @@
 ; - poll       (any POSIX os)
 ; - epoll      (linux >= 2.5.44)
 ; Default Value: not set (auto detection)
-;events.mechanism = epoll
+events.mechanism = epoll
 
 ; When FPM is build with systemd integration, specify the interval,
 ; in second, between health report notification to systemd.
