diff -Nur /var/www/wordpress/wp-content/plugins/wp-otp/includes/class-wp-otp-user-meta.php wordpress/wp-content/plugins/wp-otp/includes/class-wp-otp-user-meta.php
--- /var/www/wordpress/wp-content/plugins/wp-otp/includes/class-wp-otp-user-meta.php	2018-06-15 09:01:44.893294092 +0900
+++ wordpress/wp-content/plugins/wp-otp/includes/class-wp-otp-user-meta.php	2018-06-15 10:10:53.384196162 +0900
@@ -54,7 +54,7 @@
 	 * @since 0.1.0
 	 * @var array
 	 */
-	private static $user_meta;
+	private static $user_meta = array();
 
 	/**
 	 * User ID of the user whose meta data is managed.
